### PR TITLE
Task home work 4 math operations

### PR DESCRIPTION
**Pull request by Oleksii1985**

### CODE REVIEW

File: lesson_3/hw_math_operations.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+document.addEventListener('DOMContentLoaded', () => {
+  let firstNumber = Number(prompt('Type first number: '));
+  let secondNumber = Number(prompt('Type second number: '));
+
+  resultSummary = firstNumber + secondNumber;
```
Comment (web-scavenger) [inline lesson_3/hw_math_operations.js]: don't create variable witout key world `let\const`

File: lesson_3/hw_math_operations.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  resultSubstraction = firstNumber - secondNumber;
+  resultMultiply = firstNumber * secondNumber;
+  resultDivide = firstNumber / secondNumber;
+
+  alert(`${firstNumber} + ${secondNumber} = ${resultSummary}\n${firstNumber} - ${secondNumber} = ${resultSubstraction}\n${firstNumber} * ${secondNumber} = ${resultMultiply}\n${firstNumber} / ${secondNumber} = ${resultDivide}\n`)
```
Comment (web-scavenger) [inline lesson_3/hw_math_operations.js]: template strings have option to separate string for few lines
```suggestion
  alert(`${firstNumber} + ${secondNumber} = ${resultSummary}\n
  ${firstNumber} - ${secondNumber} = ${resultSubstraction}\n
  ${firstNumber} * ${secondNumber} = ${resultMultiply}\n
  ${firstNumber} / ${secondNumber} = ${resultDivide}\n`)
```

**Review by web-scavenger — APPROVED** [review]

File: lesson_3/hw_math_operations.js (hunk 1 above)
Comment (Oleksii1985) [reply lesson_3/hw_math_operations.js]: ok, thanks

File: lesson_3/hw_math_operations.js (hunk 2 above)
Comment (Oleksii1985) [reply lesson_3/hw_math_operations.js]: I agree, more readable

**Pull request merged**